shared/android/build.gradle

```diff
@@ -10,7 +10,7 @@ def enableV8 = KB_USEV8 == 'true'
 buildscript {
     ext {
         buildToolsVersion = "29.0.2"
-        minSdkVersion = 16
+        minSdkVersion = 29
```

Comment:
This is necessary for these experiments because the NDK Trace Async doesn't exist before 29.